app/components/govuk_component/summary_list_component/action_component.rb

```diff
@@ -1,18 +1,17 @@
 class GovukComponent::SummaryListComponent::ActionComponent < GovukComponent::Base
-  class VisuallyHiddenDefaultNilClass < NilClass; end
-
   attr_reader :href, :text, :visually_hidden_text, :attributes, :classes
 
-  def initialize(href: nil, text: 'Change', visually_hidden_text: VisuallyHiddenDefaultNilClass, classes: [], html_attributes: {})
-    if config.require_summary_list_action_visually_hidden_text && visually_hidden_text == VisuallyHiddenDefaultNilClass
+  def initialize(href: nil, text: 'Change', visually_hidden_text: false, classes: [], html_attributes: {})
+    @visually_hidden_text = visually_hidden_text
+
+    if config.require_summary_list_action_visually_hidden_text && visually_hidden_text == false
```

Comment:
What happens if this config is enabled, but someone renders the component with `nil` or `""` as the visually hidden text?

Reply:
`nil` or `""` can be used to override the requirement as there might be some places where visually hidden text isn't necessary

Reply:
Ok cool, that makes sense. 

I wasn't sure if the intention of the config setting was _we should never render this component without visually hidden text_, or if it was _please provide visually hidden text unless you're omitting it intentionally_.